src/type/staticResources.ts

```diff
@@ -0,0 +1,8 @@
+export const HowToUseTextFileName = "howtouse.md";
+export const OssCommunityInfosFileName = "ossCommunityInfos.md";
+export const PolicyTextFileName = "policy.md";
+export const OssLicensesFileName = "licenses.json";
+export const ContactTextFileName = "contact.md";
+export const QAndATextFileName = "qAndA.md";
+export const UpdateInfosFileName = "updateInfos.json";
+export const PrivacyPolicyTextFileName = "privacyPolicy.md";
```

Comment:
（めちゃ細かいですが）一貫性を考えると`Text`があったりなかったりするのは違和感あるかもです。
Textを消すか、逆に`.json`のファイルにJsonを足すかですかね。
```suggestion
export const HowToUseFileName = "howtouse.md";
export const OssCommunityInfosFileName = "ossCommunityInfos.md";
export const PolicyFileName = "policy.md";
export const OssLicensesFileName = "licenses.json";
export const ContactFileName = "contact.md";
export const QAndAFileName = "qAndA.md";
export const UpdateInfosFileName = "updateInfos.json";
export const PrivacyPolicyFileName = "privacyPolicy.md";
```
```suggestion
export const HowToUseTextFileName = "howtouse.md";
export const OssCommunityInfosFileName = "ossCommunityInfos.md";
export const PolicyTextFileName = "policy.md";
export const OssLicensesJsonFileName = "licenses.json";
export const ContactTextFileName = "contact.md";
export const QAndATextFileName = "qAndA.md";
export const UpdateInfosJsonFileName = "updateInfos.json";
export const PrivacyPolicyTextFileName = "privacyPolicy.md";
```
まあ、気が向けば･･･！

Reply:
変数名に合わせた命名だったんですけど、やっぱり統一感〜は思うところではあったんですよね
後者のJson足す方が実装する時に意識しやすいので、そっちにしようと思いますー